motr/io_nw_xfer.c

```diff
@@ -564,7 +566,8 @@ static void target_ioreq_seg_add(struct target_ioreq              *ti,
 		bvec  = &ti->ti_bufvec;
 		auxbvec = &ti->ti_auxbufvec;
 		dst_attr = ti->ti_attrbuf.b_addr;
-		goff_ivec = &ti->ti_goff_ivec;
+		if (ti->ti_goff_ivec.iv_index)
```

Comment:
This if may not be needed, check by removing this if check, does it create any issue?

Reply:
It seems this will be need as in target_ioreq_init this is getting initialized.

Reply:
If we remove it, 41motr-conf-update is not working.